.circleci/config.yml

```diff
@@ -35,7 +26,13 @@ e2eTest: &e2eTest
     - run:
         name: Run test
         command: |
-          E2E_KUBECONFIG=/e2e/kubeconfig ./e2e-harness test --test-dir=${TEST_DIR}
+          docker run --rm --network host \
+            -v $(pwd)/${TEST_DIR}:/e2e \
+            -v $(pwd):/go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
+            -e E2E_KUBECONFIG=/e2e/kubeconfig \
+            -e CIRCLE_SHA1=${CIRCLE_SHA1} \
+            -w /go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
+            quay.io/giantswarm/golang:1.12.6 go test -v -tags k8srequired ./${TEST_DIR}/
```

Comment:
Who is actually going to update all the go versions at some point?

Reply:
Good point. Quick solution could be to use tag 1, so it gets auto updated with every new 1.x.y release, at least for the next build. Additionally we can put this job/steps into a reusable orb command (either in existing architect or some new orb). @xh3b4sd @kopiczko what do you think?

Reply:
Switched to golang:1 via https://github.com/giantswarm/kubernetes-kube-state-metrics/pull/52

Reply:
I think pawel had the right idea there:
This whole block could be part of an orb, so we have a single place for this go version?
I think that should be possible.

Reply:
Yeah making it an orb is a way to go IMO. I want to do this as a part of releng SWH.